.github/workflows/ci.yml

```diff
@@ -0,0 +1,27 @@
+on: [push, pull_request]
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: ruby/setup-ruby@v1
+        with:
+          ruby-version: 3.0
+          bundler-cache: true
+      - uses: actions/setup-java@v2
+        with:
+          java-version: 11
+          distribution: adopt
+      - uses: actions/cache@v2
+        id: cache-nand2tetris
+        with:
+          path: nand2tetris
+          key: nand2tetris
+      - run: |
+          curl --location --output nand2tetris.zip 'https://drive.google.com/uc?id=1xZzcMIUETv3u3sdpM_oTJSTetpVee3KZ&export=download'
```

Comment:
If this file is in someone's google drive, wouldn't it be more future proof to just store the zip file in this repo?

Reply:
I don’t think it is, unless I’m missing something — it [used to be](https://github.com/computationclub/vm-translator/pull/3/commits/9ac682c1149b397d3dea6cad33d332fe9fbcee61#diff-6ac3f79fc25d95cd1e3d51da53a4b21b939437392578a35ae8cd6d5366ca5485L5) available there, but then they switched to hosting it on Google Drive, so I think the latter is the closest to an “official” URL that exists. 🤷🏻‍♂️

Reply:
> so I think the latter is the closest to an “official” URL that exists

Right. It looks like the last change was from 2018: 

<img width="509" alt="Screenshot 2021-07-30 at 13 33 50" src="https://user-images.githubusercontent.com/1354439/127653608-e29c626b-0a52-447a-84a0-202c5e48b5d8.png">

I'll just leave the kit here in case that url ever breaks: [nand2tetris.zip](https://github.com/computationclub/vm-translator/files/6907523/nand2tetris.zip)